Headlines/src/Controllers/TrendingCardsViewController.swift

```diff
@@ -11,7 +11,10 @@ import SDWebImage
 import Crashlytics
 import StoreKit
 
-class TrendingCardsViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource {
+class TrendingCardsViewController: UIViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f539ce350d0d725450191272ea6e28d081f1b565/Headlines/src/Controllers/TrendingCardsViewController.swift#L14"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 234 lines</td>
    </tr>
  </tbody>
</table>